bin/build_fixups.sh

```diff
@@ -32,30 +32,32 @@ else
 fi
 # --- end standard header ----
 
+OPENMP_EXTRA_EXAMPLES=$AOMP/share/openmp-extras
 # Copy examples 
-if [ -d $AOMP/examples ] ; then 
-  $SUDO rm -rf $AOMP/examples
+if [ -d $OPENMP_EXTRA_EXAMPLES ] ; then 
+  $SUDO rm -rf ${OPENMP_EXTRA_EXAMPLES}/
+else
+  $SUDO mkdir -p $OPENMP_EXTRA_EXAMPLES
```

Comment:
This mkdir should not be under the else, but always executed...otherwise you will sometimes have different behavior below depending on if openmp-extras already exists:
if openmp-extras already exists:
   examples will be copied to $AOMP/share/openmp-extras
if openmp-extras doesn't already exist:
   examples will be copied to $AOMP/share/openmp-extras/examples

Reply:
Done - mkdir is moved out of if  loop , so it will get executed always